source/textInfos/__init__.py

```diff
@@ -519,6 +519,9 @@ def collapse(self, end=False):
 		"""
 		raise NotImplementedError
 
+	def collapseAndMaybeMoveToNextParagraph(self, end: bool = False):
+		return self.collapse(end)
```

Comment:
The return statement makes me question the return type

```suggestion
	def collapseAndMaybeMoveToNextParagraph(self, end: bool = False):
		self.collapse(end)
```

or

```suggestion
	def collapseAndMaybeMoveToNextParagraph(self, end: bool = False) -> None:
		return self.collapse(end)
```